packages/graph/hash_graph/README.md

```diff
@@ -1,5 +1,39 @@
 # The HASH Graph API
 
+## Run the Graph
+
+The easiest way to run the Graph API is to use docker. Either invoke docker directly:
+
+```shell
+docker build --tag graph --file ../deployment/graph/Dockerfile .
```

Comment:
Was greeted with
`the --mount option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled`
when trying to build locally. We should probably add the ENV var in front of the build and maybe refer to https://docs.docker.com/develop/develop-images/build_enhancements/#to-enable-buildkit-builds for enabling buildkit by default